test/Ownable.js

```diff
@@ -25,15 +25,15 @@ contract("Ownable", (accounts) => {
     })
 
     it("should prevent to claimOwnership from not-pendingOwner", async () => {
-        assertFails(ownable.claimOwnership({ from: accounts[2] }), "onlyPendingOwner")
+        assertFails(ownable.claimOwnership({ from: accounts[2] }), "Returned error: VM Exception while processing transaction: revert onlyPendingOwner -- Reason given: onlyPendingOwner.")
```

Comment:
eww, here it would be better to change `assertFails` to accept a regex, then match using that

Reply:
like, if regex, try match; if string, do like now

Reply:
ok found it's fixed in monoplasma, so importing the fix from there